### ISSUE TITLE
New version update from Sketch v70

### ISSUE THREAD

**Issue by klvnchua** [duraki/SketchCrapp#41]
**Sketch version [required]**
What is the Sketch version you want support for?
70

**Additional context [optional]**
Add any other context about the request here.

**Comment by elijahtsai** [comment]
We are working on it.🤪

**Comment by evasoek** [comment]
Well, guess I'll have to wait!
Probably the reason my script says 'Can't find Sketch with that signature. Hash is invalid.'

Going to sub to this issue for updates 😬

**Comment by elijahtsai** [comment]
@evasoek , You can try 69.2 which is our latest supported version in case you need to use app before we release v70.

**Comment by Benkaa** [comment]
@any plan on supporting new m1 chip? 
just tried with 69.2, all went well but sketch is not opening anymore, keep crashing before starting.

**Comment by elijahtsai** [comment]
Hi, @Benkaa First, Yes! I and @duraki are working on new m1 chip arch application and it takes time. so stay tuned for that😄. Second, if there was possible, can you download a new v69.2 app from Sketch official website, and patch the application using our script. Remember! DO NOT open the application before the patching process complete, after the process finished you can try to open the application. If the problem still up to you, please raise a new issue and paste all the terminal information to us for more understand the problem you are facing.

**Comment by Benkaa** [comment]
Hey @elijahtsai, my bad indeed, I did open the app before patching to try if sketch was working with m1, now it looks good so far. Thanks to you and @duraki for all the hard work you're doing! 😄

**Comment by andriilive** [comment]
Offtop: any chance u guys will switch from Sketch to Framer X and Crapp it?

**Comment by elijahtsai** [comment]
@andriilive No, currently I and @duraki are both really busy with our life.

**Comment by duraki** [comment]
@andriilive that is an web-app and not a desktop app. I don't see a way we can crack it. Also, there is a free tier option you can use for personal duties.

**Comment by andriilive** [comment]
@duraki they have https://www.framer.com/desktop with .framerfx git based projects and npm. I was subscribed until rune out of budget, cause now they have only yearly options and now even can't open my projects

**Comment by elijahtsai** [comment]
Hello, Everyone we've updated the script to support up to 70.3!!! download the latest version and patch your Sketch 🥳
